File: code/packages/fsharp/ruby-parser/tests/CodingAdventures.RubyParser.Tests/RubyParserTests.fs
namespace CodingAdventures.RubyParser.Tests

open CodingAdventures.RubyParser.FSharp
open Xunit

module RubyParserTests =
    [<Fact>]
    let pingReturnsPackageName () =
        Assert.Equal("ruby-parser", RubyParser().Ping())

File: code/packages/csharp/ruby-parser/tests/CodingAdventures.RubyParser.Tests/RubyParserTests.cs
namespace CodingAdventures.RubyParser.Tests;

public sealed class RubyParserTests
{
    [Fact]
    public void PingReturnsPackageName()
    {
        Assert.Equal("ruby-parser", new CodingAdventures.RubyParser.RubyParser().Ping());
    }
}